lib/load.php

```diff
@@ -114,12 +114,18 @@ function gutenberg_is_experiment_enabled( $name ) {
 require __DIR__ . '/compat/wordpress-6.0/block-patterns.php';
 require __DIR__ . '/compat/wordpress-6.0/block-template.php';
 require __DIR__ . '/compat/wordpress-6.0/edit-form-blocks.php';
-require __DIR__ . '/experimental/register-webfonts-from-theme-json.php';
+require __DIR__ . '/experimental/class-theme-json-webfonts-handler.php';
 require __DIR__ . '/experimental/class-wp-theme-json-resolver-gutenberg.php';
+
+/*
+ * DO NOT MERGE THIS CHANGE.
+ * Don't load the Webfonts API.
+require __DIR__ . '/experimental/register-webfonts-from-theme-json.php';
 require __DIR__ . '/experimental/class-wp-webfonts.php';
 require __DIR__ . '/experimental/class-wp-webfonts-provider.php';
 require __DIR__ . '/experimental/class-wp-webfonts-provider-local.php';
 require __DIR__ . '/experimental/webfonts.php';
+ */
```

Comment:
This is a temporary workaround for testers to test this PR. Once the handler is backported to Core, then (in a separate PR), `remove_action( 'plugins_loaded', '_wp_theme_json_webfonts_handler' );` can be added to turn off the stopgate handler in Core and use the Webfonts API.